### PR TITLE
Update icub-whole-body-torque-control-faqs.md

### PR DESCRIPTION
**Pull request by misaki43**
Hello,

I don't know which precision you want, maybe only “titles” are interesting for you but I propose the following lines. It is not my final version (I have to check what I have written and correct my bad English) but I want to know what you want I write more exactly.
So, read only the title and other lines “in diagonal”.
In this document, I use "###" title to ask you some things.
I put it there since it concerns not only the iCub preliminary checks.

### CODE REVIEW

File: doc/icub-whole-body-torque-control-faqs.md [hunk 1]
```diff
@@ -6,3 +6,102 @@ Frequent Asked Questions on whole-body torque control on the iCub robot  {#icub-
...
+The configuration file for the head is on another folder:
+> $CODYCO_SUPERBUILD_ROOT/
+Take the file linked to the head and modify the _zero parameter_.
+You need to receive the acceleration values of the head to be able to calibrate it: launch a program who gives these data. Like previously, choose to the _idle mode_ in the _yarpmanager_ for the two first joints of the head. Then, turn the head until the acceleration values are near to (0;0;9) (the gravity as to be sensed only on the z axis). If x/y position are less than 0.1 it is ok.  After that, click on run and copy the encoder value on the corresponding XML file. 
+  
+To calibrate the arm:  
+Follow [this configuration](http://wiki.).
+
+
+### **I also propose to add this following part on the installation guide... But I understand if you think it is too much: the installation guide is already long so maybe it is better to don't right that. I can also create an issue with that, where I could write something like “seems I have had some trouble during the installation, I think this information can be useful to avoid them”.**
```
Comment (traversaro) [inline doc/icub-whole-body-torque-control-faqs.md]: Good idea. Given that as you told the installation guide is quite long, we can move this part to an installation tips and tricks page.

File: doc/icub-whole-body-torque-control-faqs.md [hunk 2]
```diff
@@ -6,3 +6,102 @@ Frequent Asked Questions on whole-body torque control on the iCub robot  {#icub-
...
+* To be able to clean up easily all the project, we have to avoid the program to be installed in the “/usr” folder and also avoid the libraries installation to be in the “/lib” folder. To do that, follow these recommendations:  
+1. Create a folder like “software” in your home directory to install all programs in this folder.  
+2. When you download the file to install a program, create a “build” directory (where you create the program with '”ccmake ../” e.g. ~/software/$yourProgram/build) and also a “libraries” directory (where the libraries will be installed thanks to the cmake option “-DCMAKE_INSTALL_PREFIX=~/
+3. After the “make”, don't do “make install” that install this program in the /usr and /lib folder (thus, the installation is only done in the current folder).  
+4. Configure some variables that describe where these programs and libraries are installed. During this installation, we guide you to configure them. For example:    
+>export LD_LIBRARY_PATH=$LD_LIBRARY_
+export PATH=$PATH:~/software/$
+
+### Finally, I propose these informations if you think it is interesting.
```
Comment (traversaro) [inline doc/icub-whole-body-torque-control-faqs.md]: This definitively fits this document.

File: doc/icub-whole-body-torque-control-faqs.md [hunk 3]
```diff
@@ -6,3 +6,102 @@ Frequent Asked Questions on whole-body torque control on the iCub robot  {#icub-
 ### What are the things to check on the iCub robot before performing whole-body torque control experiments? 
 
 
+# Do a complete update of information (before the calibration)
+
+## Usual update before processing
+First, update the new version of yarp/icub_main/codyco-
+Then, if there are some updates, you have to build them (make) on all the computers (because builds are not shared) with the usual:  
+> cd build
```
Comment (traversaro) [inline doc/icub-whole-body-torque-control-faqs.md]: If you enclose the bash code blocks with three tildes before and after, you get a nice rendering of the code, as in:
~~~
cmake .
make 
~~~
To discover more on this, see https://guides.github.com/features/mastering-markdown/ .

**Comment by traversaro** [comment]
Thanks @misaki43 , I think what you did is quite good, I'll probably move the installation-related part to another document and rename this one to something that fits more the document ("Step to prepare an iCub for whole body torque control experiments").

**Comment by misaki43** [comment]
Your welcome, wait before moving it, I have to correct my bad English before.

**Comment by traversaro** [comment]
:+1:

**Comment by traversaro** [comment]
@misaki43 I never noticed the english correction! People from our lab will get in charge of merging this, thanks again!

**Comment by traversaro** [comment]
@gabrielenava @pi-q @fjandrad 
Can you take care of cleaning this document such that it can be added to the codyco documentation? 
To check out how to reproduced the documentation on your pc, check out the latest section in http://wiki.icub.org/codyco/dox/html/index.html .

**Comment by traversaro** [comment]
@DanielePucci closing this PR given the lack of interest, but I still think it is a valuable piece of documentation.

**Comment by DanielePucci** [comment]
@pi-q and @gabrielenava can take care of cleaning English of the document.

Deadline attempt: End of September

**Comment by pi-q** [comment]
I will be happy to have a look, but I don't have writing access to the file. How should I proceed?

**Comment by traversaro** [comment]
Feel free to copy the file in your fork, thanks!

**Comment by traversaro** [comment]
Moved to https://github.com/robotology/codyco-superbuild/pull/189 .

**Pull request closed without merging**